boards/arm/mm_swiftio/doc/index.rst

```diff
@@ -0,0 +1,230 @@
+.. _mm_swiftio:
+
+MadMachine SwiftIO
+##################
+
+Overview
+********
+
+The SwiftIO board is designed by MadMachine for use in Swift programming language. It's the world's first board dedicated designed for the modern Swift language. Zephyr will provide basic low-level capabilities for SwiftIo.
```

Comment:
Our project guidelines recommend keeping line length < 80 characters unless required (e.g., a long URL).
Here's a suggested rewrite:
```
The SwiftIO board, designed by MadMachine is the world's first board
designed with support for the modern `Swift language
<https://developer.apple.com/swift/>`_.  Zephyr provides basic
low-level capabilities for the SwiftIO board.  More information about
the board can be found at:
```

Reply:
Will Swift application run on top of Zephyr on this board?  If so or if not, that's worth mentioning here.